pypeit/tests/test_spectrographs.py

```diff
@@ -23,7 +23,7 @@ def test_keckdeimos():
     assert os.path.isfile(example_file), 'Could not find example file for Keck DEIMOS read.'
     det = 2
     #data, _ = s.load_raw_frame(example_file, det=det)
-    data, hdu, exptime, rawdatasec_img, oscansec_img = s.get_rawimage(example_file, det)
+    data, headarr, exptime, rawdatasec_img, oscansec_img = s.get_rawimage(example_file, det)
```

Comment:
Add a test for KCWI?  May not be ready for that though ...

Reply:
If you insist, I'll write some tests, but I think it's still early days until a full end-to-end reduction of KCWI data is possible.

Reply:
No, I won't insist, but if you have some tests that you could add for the code you've already got for KCWI (like just a simple instantiation of the spectrograph class), they're always welcome!